### ISSUE TITLE
New feature : tabs for tags in stream page

### ISSUE THREAD

**Issue by wyllyjon** [diaspora/diaspora#7827]
Hello all ! 

A feature that can be useful : 
As we can follow multiple tags, we can have a lot of information in the main stream that are not sorted. And maybe, they are some tags that are more important than other.
So, it can be useful to be able to add tabs for certain tags. For instance, I want #car #bikes not to be jumbled with other tags, so I create a tab only for #car, #bikes (or two tabs, one for each tag).

**Comment by Flaburgan** [comment]
You can select the tags you want to display in your stream using the left column (see the "#Followed tag" menu). Does that solve your use case?

**Comment by wyllyjon** [comment]
I can select the tags, but they are all displayed in the same stream. If I want to visualize only #car and #bikes, for example, I must scroll and search for these particular tags in my stream (because I have a lot of other tags I follow).
So, having tabs for specified tags can clarified my streams (I will have the general stream, and then, for example, a tab to follow #bikes and #car, a tab to follow #diy, a tab to follow #Paris, etc.)

**Comment by Flaburgan** [comment]
Oh okay, so in my opinion, this is a duplicate of https://github.com/diaspora/diaspora/issues/5221 can you please confirm?

**Comment by wyllyjon** [comment]
No. The #5221 ask for a advanced filter in the stream results.
What I want is additional tabs to have additional streams with different tags than on the main stream.
Example : 
![screenshot-2018-6-13 tabs jquery ui](https://user-images.githubusercontent.com/26768253/41340523-e3e13a78-6ef7-11e8-849a-d1fe49401de1.png)

Is it more understandable with the image ?

**Comment by Flaburgan** [comment]
Yep, I get what you want and I deeply want it too. IMO all those issues (https://github.com/diaspora/diaspora/issues/2019 too for example) are linked. The idea is to be able to create custom streams based on user criteria. So "All the posts with #car and #bike" for example, but also "All the posts with #car and #bike and not #boat", or even very complex queries like "All the posts posted by wyllyjon but not by Fla with #bike or all the posts with #boat" (eh, why not? :p).
Those queries could be done in the search bar, but also "pinned" as a stream so it can be quickly accessed (probably for the left column and not from tabs but that design can change). This leads to user being able to customize what should be in the left column (maybe he never uses the activity stream and so it can be removed from the column).

I thought about that idea for years and it is the number 2 feature I want for diaspora* after the federated following tag "bug". Unfortunately I hadn't got time to work on it at the moment :( :(

**Comment by wyllyjon** [comment]
Yep, these issues are linked, but not the same.

Ok, so we will wait and see ! :-)
